kernel_gateway/gatewayapp.py

```diff
@@ -175,18 +180,26 @@ def _api_changed(self, name, old, new):
             raise ValueError('Invalid API value, valid values are jupyter-websocket and notebook-http')
 
     allow_notebook_download_env = 'KG_ALLOW_NOTEBOOK_DOWNLOAD'
-    allow_notebook_download = Bool(config=True,
-                   help="Optional API to download the notebook source code in notebook-http mode, defaults to not allow"
+    allow_notebook_download = Bool(
+        config=True,
+        help="Optional API to download the notebook source code in notebook-http mode, defaults to not allow"
```

Comment:
Nope you are correct. I just didn't scroll down to see _allow_notebook_download.